docs/application/web/get-started/tv/first-samsung-tv-app.md

```diff
@@ -20,7 +20,13 @@ Study the following instructions to help familiarize yourself with the Tizen [We
 
        ![install TV Extension](./media/install_tv_extension.png)
 
-       For more information on updating packages, see [Updating Tizen Studio](../../../tizen-studio/setup/update-sdk.md).
+The video below shows how to set up Tizen Studio and TV Extension:
```

Comment:
In the first half of the video, Visual Studio appears instead of Tizen Studio. On a menu at Visual Studio, there is a scene of launching a Tizen package manager. Tizen Package Manager is not Tizen Studio, so it seems awkward for the video to be on this page.

Reply:
Would this page be more appropriate for this video? Although it does not address the TV part.

https://docs1.stg.tizen.org/staging/1947/application/dotnet/get-started/certificates/installing-the-extension/

Reply:
Yes, it could be. But, imo, it would be better to create a page in '.NET Application > Get Started > Install the smart TV Extension'. The text could be at the level of the page below you suggested.
https://docs1.stg.tizen.org/staging/1947/application/dotnet/get-started/certificates/installing-the-extension/